documentation/UserGuide.md

```diff
@@ -60,6 +61,18 @@ If you do not provide a mapconfig.xml file, a default list of ArcGIS Online base
 
 After the first launch, the app uses the bsaemap layers that it loaded previously. If you want to reset and re-read mapconfig.xml, you can [reset the map](#reset-the-map). Alternatively, you can manually go to the Android application settings, choose Squad Leader, and choose Clear Data. Then run the app and it will read mapconfig.xml again.
 
+#### GeoMessage files and layers
+
+GeoMessage files contain GeoMessages that can be displayed on a map. GeoMessages are expressed in XML according to the [GeoMessage specification](https://github.com/Esri/geomessage-simulator-qt/blob/master/GeoMessageSpecification/GeoMessageSpecification.md). Squad Leader's built-in map configuration attempts to load the GeoMessage file /mnt/sdcard/data/coa.xml as a layer of type Mil2525CMessageLayer. If you would like that layer to display, do the following:
+
+1. Make a directory called `data` in your device's `/mnt/sdcard` directory (probably your top-level directory when using a file explorer app or USB connection from Windows).
```

Comment:
Good point. I added an explanation in 0456802bd82cec5473398360a1278e554b8abfb6. I'm using relatively recent Android devices. Squad Leader requests WRITE_EXTERNAL_STORAGE permission, which includes both reading and writing, in [AndroidManifest.xml](https://github.com/Esri/squad-leader-android/blob/master/source/SquadLeader/app/src/main/AndroidManifest.xml).